Makefile

```diff
@@ -203,12 +206,14 @@ catalog-image: $(OPM)
 	# tag the catalog img:version as latest so that continious release
 	# is possible by refering to latest tag instead of a version
 	$(CONTAINER_RUNTIME) tag $(CATALOG_IMG) $(CATALOG_IMG_LATEST)
+	$(CONTAINER_RUNTIME) tag $(CATALOG_IMG) $(CATALOG_IMG_SHA)
```

Comment:
wouldn't this translate to `podman tag observability-operator-catalog:<date> <sha-8>` ?